CHANGES.md

```diff
@@ -1,3 +1,19 @@
+# snare 0.3.0 (2020-03-06)
+
+## Breaking changes
+
+* `snare` now only searches for a configuration file at `/etc/snare/snare.conf`.
```

Comment:
Shall we add prose to talk about `-c`?

Reply:
Fixed in https://github.com/softdevteam/snare/pull/56/commits/a75528a12e3b67ebe62089490bcc421bd13180bf.